ai-platform-unified/notebooks/official/pipelines/metrics_viz_run_compare_kfp.ipynb

```diff
@@ -1,1092 +1,1335 @@
 {
```

Comment:
<p>similar comments as I added in the other notebook re: the service account treatment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>